en/docs/learn/configuring-data-sharing-for-secondary-users.md

```diff
@@ -0,0 +1,97 @@
+# Configuring Data Sharing for Secondary Users
+
+WSO2 Open Banking supports secondary user data-sharing obligations. This requirement is introduced to the Consumer Data Specification via Decision Proposal 160: Secondary Users. 
+
+The feature allows users who have been nominated as a secondary user to an account by an account holder, to share data from those secondary user accounts with third parties.
+
+An account holder can nominate secondary users for a certain account to share data from that account with third-party service providers on his behalf. Data holders should enable the nomination of secondary users. It is up to data holders to determine the norms and regulations that must be followed.
+
+The secondary users should be able to include secondary accounts in a data-sharing arrangement and account holders should have a means to withdraw data sharing of secondary accounts from such data-sharing arrangements and to be notified about the authorization, withdrawal and expiration of such data-sharing arrangements.
+
+!!! info
+    This is only available as a WSO2 Update from **WSO2 Open Banking Identity Server CDS Toolkit Level 1.0.0.8** onwards. For more information on updating, see [Getting WSO2 Updates](../install-and-setup/setting-up-servers.md#getting-wso2-updates).
+
+## Integrating with the bank back-end
+
+### Secondary Account Update API
+
+The Secondary Account Update API is defined below.
+
+The bank needs to invoke this endpoint to update the open banking solution regarding status changes happening on secondary user accounts. This endpoint persists the received secondary user-related information in the `OB_ACCOUNT_METADATA` table. When persisting the `accountId` of the secondary account and `secondaryUserId` parameters are used as a combined key and metadata will be stored as key-value pairs.
```

Comment:
lets correct to "user related"

Reply:
Lets capitalize as "accountID" and "secondaryUserID"

Reply:
Fixed via https://github.com/wso2/docs-ob-toolkit-cds/pull/78/commits/cae48427b88d4c488acaa5414de54c8e21565690.